services/slam/fake/data_loader.go

```diff
@@ -0,0 +1,145 @@
+package fake
+
+import (
+	"encoding/json"
+	"fmt"
+	"image"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/geo/r3"
+	"github.com/pkg/errors"
+	"go.viam.com/utils"
+	"go.viam.com/utils/artifact"
+
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/spatialmath"
+	rdkutils "go.viam.com/rdk/utils"
+	"go.viam.com/rdk/vision"
+)
+
+type pose struct {
+	X float64 `json:"x"`
+	Y float64 `json:"y"`
+	Z float64 `json:"z"`
+}
+
+type poseInFrame struct {
+	Pose pose `json:"pose"`
+}
+
+type quat struct {
+	Imag float64 `json:"imag"`
+	Jmag float64 `json:"jmag"`
+	Kmag float64 `json:"kmag"`
+	Real float64 `json:"real"`
+}
+
+type extra struct {
+	Quat quat `json:"quat"`
+}
+
+type position struct {
+	Pose  poseInFrame `json:"pose"`
+	Extra extra       `json:"extra"`
+}
+
+// type positionNew struct {
+// 	Pose               pose   `json:"pose"`
+// 	ComponentReference string `json:"component_reference"`
+// 	Extra              extra  `json:"extra"`
+// }
+
+const (
+	internalStateTemplate = "%s/internal_state/internal_state_%d.pbstream"
+	maxDataCount          = 16
+	pcdTemplate           = "%s/pointcloud/pointcloud_%d.pcd"
+	jpegTemplate          = "%s/image_map/image_map_%d.jpeg"
+	positionTemplate      = "%s/position/position_%d.json"
+	// positionNewTemplate   = "%s/position_new/position_%d.json".
+)
+
+func fakeGetMap(datasetDir string, slamSvc *SLAM, mimeType string) (string, image.Image, *vision.Object, error) {
+	var err error
+	var img image.Image
+	var vObj *vision.Object
+
+	switch mimeType {
+	case rdkutils.MimeTypePCD:
+		path := filepath.Clean(artifact.MustPath(fmt.Sprintf(pcdTemplate, datasetDir, slamSvc.count())))
+		slamSvc.logger.Debug("Reading " + path)
+
+		f, err := os.Open(path)
+		if err != nil {
+			return "", nil, nil, err
+		}
+		defer utils.UncheckedErrorFunc(f.Close)
+
+		pc, err := pointcloud.ReadPCD(f)
+		if err != nil {
+			return "", nil, nil, err
+		}
+
+		vObj, err = vision.NewObject(pc)
+		if err != nil {
+			return "", nil, nil, err
+		}
+
+	case rdkutils.MimeTypeJPEG:
```

Comment:
[q] Why does this still exist when the orthographic camera should be using the pcd to create an image?

Reply:
So that I can compare the behavior of the orthographic camera to the carto image data to confirm that the three.js implementation draws the map & places the robot marker in the same way that carto does.

This can be deleted after we remove the GetMap endpoint.